src/main/java/uk/gov/companieshouse/insolvency/data/config/ApplicationConfig.java

```diff
@@ -23,9 +37,32 @@ EnvironmentReader environmentReader() {
         return new EnvironmentReaderImpl();
     }
 
+    @Bean
+    @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+    public InternalApiClient internalApiClient() {
+        return ApiSdkManager.getPrivateSDK();
+    }
+
+    /**
+     * mongoCustomConversions.
+     * @return MongoCustomConversions.
+     */
     @Bean
     public MongoCustomConversions mongoCustomConversions() {
-        return new MongoCustomConversions(List.of(insolvencyConverter));
+        ObjectMapper objectMapper = mongoDbObjectMapper();
+        return new MongoCustomConversions(List.of(new CompanyInsolvencyConverter(objectMapper),
```

Comment:
We also need to have converters for Enums to store lower case value instead of upper case, I push Enum converters for reference into PoC branch

Reply:
Changes done